docs/pages/enroll-resources/database-access/enroll-aws-databases/aws-memorydb.mdx

```diff
@@ -0,0 +1,206 @@
+---
+title: Database Access with Amazon MemoryDB
+description: How to configure Teleport database access with Amazon MemoryDB
```

Comment:
Should we use upper case here?

```suggestion
description: How to configure Teleport Database Access with Amazon MemoryDB
```

Reply:
We moved away from "[Resource] Access" feature terminology in our messaging guide a couple of years ago

Reply:
Sorry, I was confused by the "Database Access" in title, but now I see this is just a way it is capitalised.